spec/support/webmock.rb

```diff
@@ -1,13 +1,13 @@
 require 'webmock/rspec'
 
-def body_mock
-  ["disease"]
-end
-
-RSpec.configure do |config|
-  config.before do
-    stub_request(:get, "http://www.who.int/topics/infectious_diseases/factsheets/en/").
-      with(:headers => {'Accept'=>'*/*', 'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'User-Agent'=>'ruby'}).
-      to_return(:status => 200, :body => body_mock, :headers => {})
-  end
-end
\ No newline at end of file
+# def body_mock
+#   ["disease"]
+# end
+#
+# RSpec.configure do |config|
```

Comment:
We can remove it if I need to put some webmock config I can create it again. I just commented out to disable this call.